src/xrpld/app/tx/detail/VaultDeposit.cpp

```diff
@@ -80,17 +85,23 @@ VaultDeposit::preclaim(PreclaimContext const& ctx)
 
     if ((vault->getFlags() & tfVaultPrivate) && account != vault->at(sfOwner))
```

Comment:
`STObject::isFlag()` is a nice helper function that abstracts the `getFlags() & tfWhatever` check. It's not the best name. Probably should have been called `isFlagSet`. But I like it because it's more concise and precise. Consider switching to it.

```suggestion
    if ((vault->isFlag(tfVaultPrivate)) && account != vault->at(sfOwner))
```